patterns/pricing.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Title: Pricing
+ * Slug: twentytwentyfour/pricing
+ * Categories: call-to-action
+ */
+
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|40","left":"var:preset|spacing|40","right":"var:preset|spacing|40"}}},"backgroundColor":"base-2","layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide has-base-2-background-color has-background" style="padding-top:var(--wp--preset--spacing--50);padding-right:var(--wp--preset--spacing--40);padding-bottom:var(--wp--preset--spacing--40);padding-left:var(--wp--preset--spacing--40)"><!-- wp:heading {"textAlign":"center","fontSize":"x-large"} -->
+	<h2 class="wp-block-heading has-text-align-center has-x-large-font-size"><?php echo esc_html_x( 'Our Services', 'Heading for pricing pattern', 'twentytwentyfour' ); ?></h2>
+	<!-- /wp:heading -->
+
+	<!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
```

Comment:
Why is this inner group that wraps the paragraph needed?

Reply:
Content width is 620px. The design has this paragraph on one line at 721px wide. Without the group, the paragraph wraps onto a second line.